community/hangouts/index.md

```diff
@@ -32,17 +32,22 @@ The Monero community is diverse and varied. We come from all over, but we defini
                         <div class="info-block">
                             <div class="row center-xs">
                                 <div class="col">
-                                    <h2>Slack</h2>
+                                    <h2>Workgroup Resources</h2>
                                 </div>
                             </div>
                             <div class="row center-xs">
-                                <p>Not everyone likes to get on the IRC, and for those people we have Slack with relays to IRC channels.
-                                It has also got a few channels of its own.</p>
+                                <p>In an effort to support organic workgroups, Monero has several resources that the community can use to meet and plan projects. Mattermost even has relays into the most popular Monero-related IRC channels.</p>
                             </div>
                             <div class="row relays around-xs">
-                                <div class="col-md-8 col-sm-8 col-xs-8">
+                                <div class="col-md-4 col-sm-4 col-xs-4">
                                     <p><a href="https://monero.slack.com/" class="btn-link btn-fixed">Slack</a></p>
```

Comment:
Since we decided to migrate to MatterMost, I think we should remove every reference to Slack on the website.
 Referring [Issue #81 in meta](https://github.com/monero-project/meta/issues/81#issuecomment-332076651)

Reply:
Afaik, they are waiting for the Taiga/Mattermost integration before they are heavily marketed.

Reply:
If the goal is to slowly replace Slack, I think we should start removing every reference to it to avoid new users, when everything will be integrated we can start asking people to migrate to MM. I don't see the point in keeping the reference to Slack when will be soon replaced, this will just generate confusion, but this is just my thought.